src/main/kotlin/at/bitfire/dav4jvm/property/push/ContentUpdate.kt

```diff
@@ -0,0 +1,56 @@
+package at.bitfire.dav4jvm.property.push
+
+import at.bitfire.dav4jvm.Property
+import at.bitfire.dav4jvm.PropertyFactory
+import at.bitfire.dav4jvm.XmlUtils.propertyName
+import at.bitfire.dav4jvm.property.webdav.SyncLevel
+import at.bitfire.dav4jvm.property.webdav.SyncToken
+import org.xmlpull.v1.XmlPullParser
+
+/**
+ * Represents a [NS_WEBDAV_PUSH]`:content-update` property.
+ *
+ * Experimental! See https://github.com/bitfireAT/webdav-push/
+ */
+data class ContentUpdate(
+    val syncLevel: SyncLevel? = null,
```

Comment:
I had to change the `sync-level` (1 or infinite) of `content-update` to `depth` (0, 1 or infinite) again (see updated draft). A content update with depth 0 means that the property itself changes or is removed.